app/views/courses/TeacherClassView.coffee

```diff
@@ -106,6 +106,8 @@ module.exports = class TeacherClassView extends RootView
 
     if options.readOnly
       @state.set('readOnly', options.readOnly)
+    if options.vue
```

Comment:
There is some discussion about renaming the option `vue` in the parent PR, to keep in mind while merging

Reply:
Thank you! Saw this during rebasing and updated.